spec/features/create_mediated_page_request_spec.rb

```diff
@@ -7,9 +7,11 @@
       visit new_mediated_page_path(item_id: '1234', origin: 'SPEC-COLL', origin_location: 'STACKS')
       click_link "I don't have a SUNet ID"
 
+      expect(page).to have_field('Library ID', type: 'text')
       expect(page).to have_field('Name', type: 'text')
       expect(page).to have_field('Email', type: 'email')
       expect(page).to have_css('a', text: '‹ Go back (show the login option)')
+      expect(page).to have_css('input#mediated_page_user_attributes_library_id', :focus)
```

Comment:
Try commenting our the javascript that applies the focus and see if this test still passes.  I have a feeling it will pass regardless because `:focus` is not an appropriate filter for `have_css`.